operatorconfig/driverconfig/common/k8s-1.31-values.yaml

```diff
@@ -21,5 +21,5 @@ images:
   sdcmonitor: dellemc/sdc:4.5.2.1
   # "images.metadataretriever" defines the container images used for csi metadata retriever
   metadataretriever: dellemc/csi-metadata-retriever:v1.8.0
-  # "images.csiReverseProxy" defines the container images used for reverse-proxy
-  csiReverseProxy: dellemc/csipowermax-reverseproxy:v2.11.0
+  # "images.csiReverseProxy" defines the container images used for reverse-proxy by default
+  csiReverseProxy: dellemc/csipowermax-reverseproxy:v2.10.0
```

Comment:
Why the reverse proxy version is downgraded? Is it intentional?

Reply:
As of now `2.11.0` is not released if we maintain that in the yamls then the installation will fail. Once we release the latest version we need to bump up here